flamedisx/er_nr_base.py

```diff
@@ -217,15 +645,11 @@ def rate_nq(self, nq_1d, data_tensor, ptensor):
 
         return tf.reduce_sum(p_nq_e * rate_e[:, o, :], axis=2)
 
-    @staticmethod
-    def penning_quenching_eff(nph, eta=8.2e-5 * 3.3, labda=0.8 * 1.15):
-        return 1. / (1. + eta * nph ** labda)
-
     def _simulate_nq(self, energies):
         # OK to use None, simulator has set defaults
         work = self.gimme('work', data_tensor=None, ptensor=None, numpy_out=True)
         lindhard_l = self.gimme('lindhard_l',
                                 bonus_arg=energies,
                                 data_tensor=None, ptensor=None,
                                 numpy_out=True)
-        return stats.poisson.rvs(energies * lindhard_l / work)
+        return stats.poisson.rvs(energies * lindhard_l / work)
```

Comment:
Why does your editor always remove the newlines at the end of files?

Reply:
I think it's just me :-)